docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -16,38 +16,51 @@ We will start with a single cluster and move to multiple clusters to illustrate
 
 ## Installation and Setup
 * Clone this repo locally 
-* Setup a `./controller-config.env` file in the root of the repo with the following key values
+* Setup a `./controller-config.env` env-var file in the root of the repo with the following keys. Fill in your own values as appropriate. You will need access to a domain or subdomain in Route 53 in AWS:
 
-    ```bash
-    # this sets up your default managed zone
-    AWS_DNS_PUBLIC_ZONE_ID=<AWS ZONE ID>
-    # this is the domain at the root of your zone (foo.example.com)
-    ZONE_ROOT_DOMAIN=test.hcpapps.net # replace this with your root domain
-    LOG_LEVEL=1
-    ```   
+  ```bash
+  AWS_DNS_PUBLIC_ZONE_ID=Z01234567US0IQE3YLO00
+  ZONE_ROOT_DOMAIN=jbloggs.hcpapps.net
+  LOG_LEVEL=1
+  ```
+
+  | Env Var                  | Example Value           | Description                                           |
+  |--------------------------|-------------------------|-------------------------------------------------------|
+  | `ZONE_ROOT_DOMAIN`       | `jbloggs.hcpapps.net`   | Hostname for the root Domain                          |
+  | `AWS_DNS_PUBLIC_ZONE_ID` | `Z01234567US0IQE3YLO00` | AWS Route 53 Zone ID for specified `ZONE_ROOT_DOMAIN` |
+  | `LOG_LEVEL`              | `1`                     | Log level for the Controller                          |
 
 * setup a `./aws-credentials.env` with credentials to access route 53
 
-    For example:
+  For example:
+
     ```bash
     AWS_ACCESS_KEY_ID=<access_key_id>
     AWS_SECRET_ACCESS_KEY=<secret_access_key>
     AWS_REGION=eu-west-1
     ```
 
+  | Env Var                 | Example Value          | Description                                                    |
+  |-------------------------|------------------------|----------------------------------------------------------------|
+  | `AWS_ACCESS_KEY_ID`     | `AKIA1234567890000000` | Access Key ID, with access to resources in Route 53            |
+  | `AWS_SECRET_ACCESS_KEY` | `Z01234567US0000000`   | Access Secret Access Key, with access to resources in Route 53 |
+  | `AWS_REGION`            | `eu-west-1`            | AWS Region                                                     |
+
+
 * We're going to use an environment variable, `MGC_SUB_DOMAIN`, throughout this walkthrough. Simply run the below in each window you create:
```

Comment:
No harm in reminding of this when T2 is opened later in the guide.
Can be in a follow up PR since you're planning on making further changes around the getting started experience.